specs/PrintToPdf.md

```diff
@@ -0,0 +1,325 @@
+                            MessageBox(nullptr, formattedMessage.c_str(),
+                                L"Print to PDF Completed", MB_OK);
+                        };
+                        m_appWindow->RunAsync([showDialog]() { showDialog(); });
+                        return S_OK;
+                    })
+                    .Get()));
+        }
+    }
+}
+```
+
+```c#
+// Prints current page to PDF using the default path and settings. If requested,
+// the orientation is changed to landscape.
+async void PrintToPdfCmdExecuted(object target, ExecutedRoutedEventArgs e)
+{
+    CoreWebView2PrintSettings printSettings =
+        WebViewEnvironment.CreatePrintSettings();
+
+    string orientationString = e.Parameter.ToString();
+    if (printSettings != null && orientationString == "Landscape")
+    {
+        printSettings.Orientation =
+            CoreWebView2PrintOrientation.Landscape;
+    }
+
+    Microsoft.Win32.SaveFileDialog openFileDialog =
+        new Microsoft.Win32.SaveFileDialog();
+    openFileDialog.InitialDirectory = "C:\\";
+    openFileDialog.Filter = "Pdf Files|*.pdf";
+    Nullable<bool> result = openFileDialog.ShowDialog();
+    if (result == true) {
+        string resultFilePath = await webView.CoreWebView2.PrintToPdfAsync(
+            openFileDialog.FileName, printSettings);
+        string message = (resultFilePath == string.Empty) ?
+            "Print to PDF failed" : "ResultFilePath: ";
+        MessageBox.Show(this, message + resultFilePath,
+                        "Print To PDF Completed");
+    }
+}
+```
+# Remarks
+
+# API Notes
+
+
+# API Details
+```
+interface ICoreWebView2_4;
+interface ICoreWebView2Environment5;
+interface ICoreWebView2PrintSettings;
+interface ICoreWebView2PrintToPdfCompletedHandler;
+
+
+/// The orientation for printing, used by the `Orientation` property on
+/// `ICoreWebView2SPrintSettings`. Currently only printing to PDF
+/// is supported.
+[v1_enum]
+typedef enum COREWEBVIEW2_PRINT_ORIENTATION
+{
+  /// Print the page(s) in portrait orientation.
+  COREWEBVIEW2_PRINT_ORIENTATION_PORTRAIT,
+
+  /// Print the page(s) in landscape orientation.
+  COREWEBVIEW2_PRINT_ORIENTATION_LANDSCAPE,
+} COREWEBVIEW2_PRINT_ORIENTATION;
+
+[uuid(9eae81c3-fe02-4084-b475-903a4ed9252e), object, pointer_default(unique)]
+interface ICoreWebView2_4 : IUnknown {
+  /// Print the current page to PDF asynchronously with the provided settings.
+  /// See `ICoreWebView2PrintSettings` for description of settings.
+  ///
+  /// Use `resultFilePath` to specify the path to the PDF file. The host should
+  /// provide an absolute path, including file name. If the path
+  /// points to an existing file, the file will be overwritten. If the path is
+  /// not valid, the method fails with `E_INVALIDARG`.
+  ///
+  /// The async `PrintToPdf` operation completes when the data has been written
+  /// to the PDF file. At this time the
```

Comment:
Likely truncated or deleted because we run in the same process. What about closing the webview2 during printing? 
* Please investigate and document.
* Consider updating sample app to prompt user not to close during async operation.